packages/protocol-parser/src/decodeDozerField.ts

```diff
@@ -0,0 +1,24 @@
+import { AbiType } from "@latticexyz/config";
+import {
+  ArrayAbiType,
+  SchemaAbiTypeToPrimitiveType,
+  arrayToStaticAbiType,
+  schemaAbiTypeToDefaultValue,
+} from "@latticexyz/schema-type/internal";
+
+export function decodeDozerField<abiType extends AbiType>(
```

Comment:
chatted IRL, should move these into `store-sync/dozer` since they're not store/world protocol things